### PR TITLE
report(bootup-time): fix learn more link

### PR DESCRIPTION
**Pull request by underbyte**

### CODE REVIEW

File: lighthouse-core/test/results/sample_v2.json [hunk 1]
```diff
@@ -1101,7 +1101,7 @@
       "scoreDisplayMode": "numeric",
       "name": "bootup-time",
       "description": "JavaScript boot-up time",
-      "helpText": "Consider reducing the time spent parsing, compiling, and executing JS. You may find delivering smaller JS payloads helps with this. [Learn more](https://developers.google.com/web/lighthouse/audits/bootup).",
+      "helpText": "Consider reducing the time spent parsing, compiling, and executing JS. You may find delivering smaller JS payloads helps with this. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/bootup).",
```
Comment (patrickhulce) [inline lighthouse-core/test/results/sample_v2.json]: @paulirish yeah ignore help text is a good call for our golden file assertions :)

**Comment by patrickhulce** [comment]
@underbyte mind your 🖊 on the CLA to make the 🕴 happy? :)

**Comment by 4194sunny** [comment]
Like

**Comment by underbyte** [comment]
@patrickhulce The CLA site says I have one. Do I need to configure it in github or anything?

**Comment by patrickhulce** [comment]
oh probably just email issue then with git config thanks! 👍

**Pull request merged**